ante/ante.go

```diff
@@ -9,22 +12,23 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/auth/ante"
-	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
 
 	gaiaerrors "github.com/cosmos/gaia/v18/types/errors"
-	gaiafeeante "github.com/cosmos/gaia/v18/x/globalfee/ante"
 )
 
+// UseFeeMarketDecorator to make the integration testing easier: we can switch off its ante and post decorators with this flag
```

Comment:
This is a workaround to make the integration tests based on IBC testing framework work without the need to override the `SendMsgs()` function, since this would also require changes to the ICS repository to modify the `CCVTestSuite`. Also, our non-determinism tests were failing and we are unable to provide the tx fees to the framework used, so we opted for this work-around. For the override of the `SendMsgs()` see the `FeeMarketTestSuite`.